date/rentals.json

```diff
@@ -0,0 +1 @@
+[{"json_class":"Rental","date":"21/03","person":{"json_class":"Teacher","id":167,"name":"Vicente","age":30,"specialization":"vet"},"book":{"json_class":"Book","title":"Harry Potter","author":"Rowling"}},{"json_class":"Rental","date":"21/03","person":{"json_class":"Student","id":422,"name":"Gabriel","age":20,"parent_permission":true,"classroom":null},"book":{"json_class":"Book","title":"The Lord of the Rings","author":"Tolkien"}}]
```

Comment:
- [ ] It appears you are not making use of the files within the `date` directory. When I create books, people, and rentals, the files are saved in JSON files within the root path. Kindly remove this file and all the ones within the `date` folder to keep your code clean.